core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/RowParser.java

```diff
@@ -464,6 +457,18 @@ private String asValidatedString(
     return s;
   }
 
+  public enum NumberBounds {
+    POSITIVE,
+    NON_NEGATIVE,
+    NON_ZERO,
+  }
+
+  @FunctionalInterface
+  public interface EnumCreator<E> {
+
+    E convert(int t);
+  }
+
```

Comment:
This is a refactoring, possibly initiated by Idea when you allow it to rearrange entries. Please restore the previous position since refactoring should not be mixed with functional changes.